Sources/IRGen/Expression/IRExpression.swift

```diff
@@ -8,6 +8,20 @@
 import AST
 import Lexer
 
+struct GeneratedCode {
+  var preamble: String
+  var expression: String
```

Comment:
`preamble` and `expression` should really be constants. This is in-line with other support structures in the compiler and makes the semantics very clear. If someone wanted mutate a `GeneratedCode` or `ExpressionFragment` then they would have to make an explicit copy (we want this)